### ISSUE TITLE
Multiscreen support

### ISSUE THREAD

**Issue by Curran919** [DaltonLens/DaltonLens#9]
I noticed the colorpicker only works on the primary monitor. I can see that having the secondary monitor interactable while the colorpicker is deployed only on the primary monitor can be more of a feature than a bug. However, would it be possible for the colorpicker to be active on the screen that the cursor is on when the hotkey is pressed?

**Comment by nburrus** [comment]
Yes, that would be ideal. It's not like this right now only because properly handling monitor (possibly with different DPI) is a bit tricky to do, and I only have one :) But I'll keep that in the TODO list.

**Comment by Curran919** [comment]
Of course, with my limited desktop knowledge, I can imagine it being a pain with the different DPIs or even different resolutions so the total desktop is not even rectangular anymore. That's why I thought it would be 10x easier to implement using one screen at a time.

More importantly though, _how do you work with only one screen!_

**Comment by nburrus** [comment]
Yes, but even with just one screen at a time we have to support a dynamic change of the DPI for fonts, etc. Not crazy hard, but the solution will probably look like this: https://github.com/ocornut/imgui/issues/1676#issuecomment-452907351 .

> More importantly though, how do you work with only one screen!

I stopped using multiple monitors after I got my 27 inches one a few years ago :)

**Comment by anna-oake** [comment]
+1 this is really needed.

**Comment by zimonitrome** [comment]
This project is probably a bit abandoned but I just found it and love it. A big bump up from "colorblind assistant".
Would love multiscreen support.

**Comment by nburrus** [comment]
Thanks for the comment, my sabbatical ended and I haven't been able to do much more since then, but let's see I can do something about multiscreen soon.

**Comment by nburrus** [comment]
didn't happen "soon" but finally managed to spend some time on this and just did an early release with multi-screen support. Still need to update the macOS app store. I haven't tested extensively, but it seems to work reasonably well for me on macOS / Windows and Linux.
